Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/zepben/dss-capi-builder:latest
+
+RUN mkdir -p /app /outputs
+WORKDIR /app
+
+COPY . .
+
+RUN make rmqpush && ./build/build_linux_x64.sh && cp /app/lib/linux_x64/*.so /outputs
```

Comment:
Thought: So I'm assuming our goal with building a container here is to package the libraries, i.e the contents of the `/outputs` directory. Maybe we should clear the `/app` directory then while at it, to save space of the resulting container?
:thinking: